horace_core/sqw/file_io/sqw_formats_factory.m

```diff
@@ -30,29 +30,29 @@
     %                       used to initialize another accessor.
     %
     %
-    % $Revision:: 1759 ($Date:: 2020-02-10 16:06:00 +0000 (Mon, 10 Feb 2020) $)
     %
     properties(Access=private) %
         % List of registered file accessors:
         % Add all new file readers which inherit from sqw_file_interface and dnd_file_interface
         % to this list in the order of expected frequency of their appearance.
-        supported_accessors_ = {faccess_sqw_v3(),faccess_sqw_v3_2(),...
+        supported_accessors_ = {faccess_sqw_v3_3(),faccess_sqw_v3(),...
```

Comment:
is ordering important in this list? `3.3, 3, 3.2, 2, dnd2, ...`

Reply:
algorithm checks the file formats in the order of probability to find appropriate. First check for the current, second for previous. 3.2 is experimental for indirect and used for Mashroom only (may be useful in SNS ?) Its probably worth to modify selection to add some memory of most frequent recently found, but it may be overkill.